### PR TITLE
CNCF Sandbox acceptance blog post

### PR DESCRIPTION
**Pull request by sjwoodman**
Blog post describing Strimzi moving to CNCF

### CODE REVIEW

File: _posts/2019-08-29-cncf.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+From the core Kubernetes project to exciting early stage projects such as CloudEvents we feel Strimzi is in esteemed company.
+
+<!--more-->
+
+Strimzi provides Operators to deploy and manage the components of [Apache Kafka](https://kafka.apache.org/) on Kubernetes.
+This includes the Kafka brokers, Apache ZooKeeper, MirrorMaker and Kafka Connect.
+Having Kubernetes-native APIs makes it easy to manage Kafka topics and users, and there are [multiple ways](https://strimzi.io/2019/04/17/accessing-kafka-part-1.html) to enable secure access to Kafka from outside the Kubernetes cluster.
+Recent additions to the project have included an [HTTP bridge](https://github.com/strimzi/strimzi-kafka-bridge) to enable additional protocol support; improved support for [scaling the persistent storage](https://strimzi.io/2019/07/08/persistent-storage-improvements.html) used by the Kafka brokers and monitoring using Prometheus.
+
+Since its inception in 2017 Strmizi has become a production ready mechanism for running Apache Kafka on Kubernetes.
```
Comment (ppatierno) [inline _posts/2019-08-29-cncf.md]: typo "Strimzi"

File: _posts/2019-08-29-cncf.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Initially begun by three developers, the project now has significantly more than that contributing to making it a success from organisations such as Red Hat, Lightbend and others.
```
Comment (ppatierno) [inline _posts/2019-08-29-cncf.md]: link to the companies web site?

File: _posts/2019-08-29-cncf.md (hunk 1 above)
Comment (sjwoodman) [reply _posts/2019-08-29-cncf.md]: Good spot!

File: _posts/2019-08-29-cncf.md (hunk 2 above)
Comment (sjwoodman) [reply _posts/2019-08-29-cncf.md]: done

File: _posts/2019-08-29-cncf.md [hunk 3]
```diff
@@ -24,7 +24,11 @@ We hope that membership of the CNCF will allow many more users to adopt Apache K
 
 The Strimzi community is friendly and driven by the desire to further Open Source principles - we are always open to new contributions.
 Please do get involved, not just through code; documentation improvements, bug reports and blog posts are all appreciated.
-You can download the latest version of Strimzi [here](https://strimzi.io/downloads/) and say hello on [Slack](https://join.slack.com/t/strimzi/shared_invite/enQtMzU2Mjk3NTgxMzE5LTYyMTUwMGNlMDQwMzBhOGI4YmY4MjhiMDgyNjA5OTk2MTFiYjc4M2Q3NGU1YTFjOWRiMzM2NGMwNDUwMjBlNDY).
+
+As Strimzi moves to the CNCF we are taking the opportunity to make a few changes to the communication methods we use. 
+Firstly, we will be moving to a [CNCF hosted Slack channel](https://cloud-native.slack.com/messages/CMH3Q3SNP) which, as an enterprise version, has unlimited users and message history.
```
Comment (ppatierno) [inline _posts/2019-08-29-cncf.md]: isn't it better to provide this link https://slack.cncf.io/ and then saying that we have the #strimzi channel there?

File: _posts/2019-08-29-cncf.md [hunk 4]
```diff
@@ -24,7 +24,11 @@ We hope that membership of the CNCF will allow many more users to adopt Apache K
 
 The Strimzi community is friendly and driven by the desire to further Open Source principles - we are always open to new contributions.
 Please do get involved, not just through code; documentation improvements, bug reports and blog posts are all appreciated.
-You can download the latest version of Strimzi [here](https://strimzi.io/downloads/) and say hello on [Slack](https://join.slack.com/t/strimzi/shared_invite/enQtMzU2Mjk3NTgxMzE5LTYyMTUwMGNlMDQwMzBhOGI4YmY4MjhiMDgyNjA5OTk2MTFiYjc4M2Q3NGU1YTFjOWRiMzM2NGMwNDUwMjBlNDY).
+
+As Strimzi moves to the CNCF we are taking the opportunity to make a few changes to the communication methods we use. 
+Firstly, we will be moving to a [CNCF hosted Slack channel](https://cloud-native.slack.com/messages/CMH3Q3SNP) which, as an enterprise version, has unlimited users and message history. 
+We understand that moving the Slack channel is painful for all the users of the current channel, but feel that in this case, the benefits outweigh the pain. 
+Secondly, we will be moving to a new mailing list, again hosted by CNCF. The new mailing list will be [strimzi-dev@lists.cncf.io](mailto://strimzi-dev@lists.cncf.io).
```
Comment (ppatierno) [inline _posts/2019-08-29-cncf.md]: it should be `cncf-strimzi-dev@lists.cncf.io` (so with the `cncf-` prefix) but I asked to remove the `-dev` suffix, let's see ...

File: _posts/2019-08-29-cncf.md (hunk 3 above)
Comment (sjwoodman) [reply _posts/2019-08-29-cncf.md]: done

File: _posts/2019-08-29-cncf.md (hunk 4 above)
Comment (sjwoodman) [reply _posts/2019-08-29-cncf.md]: Updated to be the correct address but will change again if the `-dev` is removed

**Comment by gunnarmorling** [comment]
Hey @sjwoodman, nice post -- and congrats to the move to CNCF!

I could imagine the following things might be interesting to readers, too:

* Why the move to a foundation in general (it speaks of better adoption, but that might warrant some expansion) and why CNCF in particular?
* What's the migration story, i.e. will there be any impact on current users, e.g. in terms of namespaces and things like that? 
* What are potential roadmap items for Strimzi's development CNCF (of course to be defined by the wider community eventually, but it might help to provide a starting point)?
* How will governance/collaboration change look like at CNCF?
* Will the same engineers continue to work on it, will there be roles such as PMC (just an example, no idea whether it exists at CNCF) be filled by them or others? Is there any mentor role to be mentioned by CNCF?

Perhaps you could touch on (some of) these?

**Comment by gunnarmorling** [comment]
> * What are potential roadmap items for Strimzi's development CNCF

I think the graduation process at CNCF might warrant some quick mention/link.

>  Is there any mentor role to be mentioned by CNCF?

I'd list the sponsors, awesome to see Matt Klein there.

**Comment by sjwoodman** [comment]
@gunnarmorling Thanks! You made some good suggestions and I've incorporated some of them into the post. I think that others, especially the roadmap and choice of CNCF warrant more space and potentially a post of their own. So we can follow up with more in the coming weeks

File: _posts/2019-08-29-cncf.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  "Strimzi Apache Kafka Operator joins the CNCF"
+date: 2019-08-29
+author: strimzi_developers
+---
+
+We are excited to announce that Strimzi is joining the Cloud Native Computing Foundation (CNCF) as a [Sandbox Project](https://www.cncf.io/sandbox-projects/).
```
Comment (tombentley) [inline _posts/2019-08-29-cncf.md]:
```suggestion
We are excited to announce that Strimzi has joined the Cloud Native Computing Foundation (CNCF) as a [Sandbox Project](https://www.cncf.io/sandbox-projects/).
```

File: _posts/2019-08-29-cncf.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+The Strimzi community is friendly and driven by the desire to further Open Source principles - we are always open to new contributions.
+Please do get involved, not just through code; documentation improvements, bug reports and blog posts are all appreciated.
+
+As Strimzi moves to the CNCF we are taking the opportunity to make a few changes to the communication methods we use. 
+Firstly, we will be moving to the #Strimzi Channel on the [CNCF hosted Slack](https://slack.cncf.io/) which, as an enterprise version, has unlimited users and message history. 
+We understand that moving the Slack channel is painful for all the users of the current channel, but feel that in this case, the benefits outweigh the pain. 
+Secondly, we will be moving to a new mailing list, again hosted by CNCF. The new mailing list will be [cncf-strimzi-user@lists.cncf.io](mailto://cncf-strimzi-user@lists.cncf.io).
+
+However, on a technical level, most users will not notice any change from the move to the CNCF.
+The release cadence of the project will be unchanged, no breaking code or license changes are planned and the development team has remained the same.
```
Comment (tombentley) [inline _posts/2019-08-29-cncf.md]:
```suggestion
The release cadence of the project will be unchanged, no breaking code or license changes are planned and the development team remains the same.
```

**Comment by sjwoodman** [comment]
@scholzj Not sure what went wrong but think I've fixed it. Also updated the publication date

**Pull request merged**